tox.ini

```diff
@@ -0,0 +1,32 @@
+[tox]
+skipsdist=True
+envlist=pycodestyle,pylint,yamllint
+
+[testenv]
+deps =
+  -cconstraints.txt
+  -raws/requirements.txt
+
+[testenv:pylint]
+deps =
+  {[testenv]deps}
+  pylint
+commands = pylint --rcfile hacking/pylint.rc aws
+
+[testenv:pycodestyle]
+deps =
+  {[testenv]deps}
+  pycodestyle
+commands = pycodestyle --config pycodestyle.ini aws
+
+[testenv:yamllint]
+deps =
+  {[testenv]deps}
+  yamllint
+commands = yamllint --config-file .yamllint aws hacking
+
+[testenv:policy]
```

Comment:
It's a good idea to provide a `description` definition so that it's available in the `tox -av` output.